community-operators/node-problem-detector/node-problem-detector.v0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,171 @@
+#! validate-crd: deploy/chart/templates/0000_30_02-clusterserviceversion.crd.yaml
+#! parse-kind: ClusterServiceVersion
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    categories: openshift optional
+    certifiedLevel: Primed
+    containerImage: registry.svc.ci.openshift.org/openshift/origin-v4.0:node-problem-detector-operator
+    createdAt: 2019/01/31
+    description: An operator to run the OpenShift Node Problem Detector
+    healthIndex: B
+    repository: https://github.com/openshift/node-problem-detector-operator
+    support: Red Hat
+  name: node-problem-detector.v0.0.1
+  namespace: openshift-node-problem-detector
+spec:
+  description: |
+      # node-problem-detector
```

Comment:
nit: Could this use the operator's display name? I think it would look better in the UI.